### ISSUE TITLE
split_key for val_dataset is test

### ISSUE THREAD

**Issue by yuming-p** [tianrun-chen/SAM-Adapter-PyTorch#116]
Hi SAM-Adapter developers,

thanks for this great repo. I see that in configs/*.yaml, split_key for val_dataset is test, instead of val, which means validation set is actually not used at all and test set is used for evaluation?
could you please explain why it is set this way?

Best,
Yuming
